### DISCUSSION TITLE
Organisation level pages ? Is this possible?

### DISCUSSION THREAD

**Discussion by joerowe** [community/community#23619] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,</p>
<p>I’d like to point a subdomain to <code>&lt;organization&gt;.github.io</code> so that I can do something like <code>docs.mydomain.com/my-repo</code>, but am getting 404s.</p>
<p>Here’s an example:</p>
<p>Suppose I have the organisation <code>foo</code>, which is at <code>foo.github.com</code> and ​I have three repos, <code>bar</code>, <code>baz</code> and <code>qux</code>.</p>
<p>I can set up CNAMES as follows:</p>
<p><code>bar.foo.com</code> → <code>foo.github.io/bar</code><br>
<code>baz.foo.com</code> → <code>foo.github.io/baz</code><br>
<code>qux.foo.com</code> → <code>foo.github.io/qux</code> and so on…</p>
<p>But this doesn’t scale well… What I’d rather do is use <code>docs.foo.com</code> as a simple CNAME to <code>foo.github.io</code>, so that I have a consistent naming convention for documentation across the organisation. E.g:</p>
<p><code>docs.foo.com/bar</code> → <code>foo.github.io/bar</code><br>
<code>docs.foo.com/baz</code> → <code>foo.github.io/baz</code><br>
<code>docs.foo.com/qux</code> → <code>foo.github.io/qux</code></p>
<p>So, I’ve set up a CNAME that points <code>docs.foo.com</code> to <code>foo.github.io</code>, but I get 404s!</p>
<p>What am I doing wrong?? It feels like this should <em>just work</em> ? I’m wondering if its maybe something to do with the source of the request not being <code>github.io</code>? Or because its a private organisation</p>

**Comment by yoannchaudet** [comment]
<p>Did <a href="https://github.community/t/pages-with-custom-subdomain-and-path/186597/2" class="inline-onebox">Pages with custom subdomain and path - #2 by yoannchaudet</a> help you already?</p>
<p>This is a supported scenario but for public Pages sites only.</p>

**Comment by joerowe** [comment]
<p>yea :frowning: i hope its on roadmap for private Pages sites</p>

**Comment by yoannchaudet** [comment]
<p>This is not on the roadmap currently.</p>
<p>Technically this cannot even be implemented in a safe way with the current web standards. If you are up for a good read, I recommend this <a href="https://github.blog/2013-04-09-yummy-cookies-across-domains/">blog post</a> from 2013 which will give you some insights as to why Private Pages are getting their own dedicated domain even when no custom domain is involved.</p>
